models/remote_log.go

```diff
@@ -0,0 +1,26 @@
+package models
+
+// RemoteLog ...
+type RemoteLog struct {
+	LogLevel string                 `json:"log_level"`
+	Message  string                 `json:"message"`
+	Data     map[string]interface{} `json:"data"`
+}
+
+// Event ...
+func (a RemoteLog) Event() string {
+	return "remote_log"
```

Comment:
As discussed, please update Data team on the data fields be loogged in Segment, event name and user ID specifically, and to let them know this new use case.

Reply:
Informed András Lendvay, he approved the event name and the user id.